docs/Contribute/ContributeWrite.md

```diff
@@ -0,0 +1,142 @@
+# Writing for BTCPay Server
+
+- [Writing for BTCPay Server](#writing-for-btcpay-server)
+  - [Why writing matters](#why-writing-matters)
+  - [Requirements and recommended software](#requirements-and-recommended-software)
+    - [Requirements](#requirements)
+    - [Recommended software](#recommended-software)
+  - [Contribute to the Software Stack](#contribute-to-the-software-stack)
+  - [Contribute to Documentation](#contribute-to-documentation)
+  - [Contribute to the Directory](#contribute-to-the-directory)
+  - [Review other's work](#review-others-work)
+  - [Get help from the community](#get-help-from-the-community)
+
+
+## Why writing matters
+
+By giving essential and concise informations on software, or creating these documentation pages to help wrap your mind around the software stack, writers contributing to BTCPayServer are as important as any other contributer.
+
+If you have writing skills, if you have a fair knowledge of the English language, then you can contribute to BTCPay Server or review the work of other contributors.
+
+Writing contributors can help in a number of places like the Software stack, the documentation or the directory for example.
```

Comment:
> Software stack

I don't think writers can contribute here? What specific things are you referring to? I think you should point people to blog instead blog.btcpayserver.org/ since that's we really need help with

Reply:
Edit, I saw what you're referring to. I really don't think writers can contribute here with writing, mostly juts proof reading and typos.

Reply:
> > Software stack
> 
> I don't think writers can contribute here? What specific things are you referring to? 

I was referring to the change in strings on the stack itself, like for example fixing typos or rewording some explanatory sentences that appear in the UI front-end.